org.godotengine.Godot3.yaml

```diff
@@ -0,0 +1,133 @@
+app-id: org.godotengine.Godot3
```

Comment:
What's the actual id? The title of the PR says org.godotengine.Godot3Sharp

Reply:
Argh, I forgot to correct that!

I'll do it now. The id's _meant_ to be `org.godotengine.Godot3Sharp`, just like in the PR title

Reply:
[I corrected it to `org.godotengine.Godot3Sharp` just now](https://github.com/flathub/flathub/pull/4595/commits/a14ecfe2e032393525755e26ba1bd9f3a53e8fcc) :+1: